llvm/test/CodeGen/RISCV/macro-fusions.mir

```diff
@@ -1,7 +1,7 @@
 # REQUIRES: asserts
```

Comment:
Let's rename the `slli_srli_...` test to `zext16...` and `zext32` and `shifted_zext32_...` to make it more consistent with other tests. I should have done this myself, but I've missed that :-<

Reply:
It would be nice to add tests that exercise logic from `checkRegisters`, but we can leave it to a different PR